test/python/primitives/test_statevector_sampler.py

```diff
@@ -619,6 +615,16 @@ def test_circuit_with_aliased_cregs(self):
             self.assertTrue(hasattr(data, creg_name))
             self._assert_allclose(getattr(data, creg_name), np.array(target[creg_name]))
 
+    def test_no_measurements(self):
```

Comment:
I suggest function name `test_no_classical_bits` or `test_no_cregs` because no measure does not raise an error if there is any classical bits.
```python
qc = QuantumCircuit(1, 1)
sampler = StatevectorSampler()
sampler.run([qc]).result()[0].data.c.get_counts()
# {'0': 1024}
```

Reply:
Thanks, I have changed it.